media/base/android/java/src/org/chromium/media/MediaPlayerBridge.java

```diff
@@ -35,12 +35,18 @@
 */
 @JNINamespace("media")
 public class MediaPlayerBridge {
-
+    /**
+    * Give the host application a chance to take over MeidaPlayer.
+    */
     public static class ResourceLoadingFilter {
         public boolean shouldOverrideResourceLoading(
-                MediaPlayer mediaPlayer, Context context, Uri uri) {
+                ExternalMediaPlayer mediaPlayer, Context context, Uri uri) {
```

Comment:
Sorry for the late response.
Is there any switch to fallback to MediaPlayer? It seems API changed. I prefer to add a new API for this.